test/acceptance/behave/README.md

```diff
@@ -21,15 +29,11 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 - Both if you are using a virtual environment or not:
   * Change to the test/acceptance/behave folder of the project.
   * Install the requirements for the acceptance tests in the virtual environment
+  * You should add "--upgrade" (if you have a previous installed version of this library)
 ```
-     pip install -r requirements.txt --allow-all-external
+     pip install [--upgrade] -r requirements.txt --allow-all-external
```

Comment:
I'm getting the following error when running the command:

```
/usr/bin/pip install: error: no such option: --allow-all-external
```

Maybe a minium version of pip is required to work?